jasminewd/index.d.ts

```diff
@@ -0,0 +1,71 @@
+// Type definitions for jasminewd2 0.1.1
```

Comment:
please add a [tslint.json](https://github.com/DefinitelyTyped/DefinitelyTyped#lint), it should catch issues like this header version being invalid.